core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -187,47 +200,74 @@ private[history] class FsHistoryProvider(conf: SparkConf) extends ApplicationHis
         }
         .flatMap { entry =>
           try {
-            Some(replay(entry, new ReplayListenerBus()))
+            Some((getModificationTime(entry).get, entry))
           } catch {
             case e: Exception =>
               logError(s"Failed to load application log data from $entry.", e)
```

Comment:
this exception message is out of date. We're not replaying the logs here anymore